### DISCUSSION TITLE
Android Project Errors

### DISCUSSION THREAD

**Discussion by Bloppl** [MonoGame/MonoGame#8286] [Android]
Greetings everyone!

I am attempting to make an Android Game with MonoGame and C#, and the MonoGame packages have warning symbols next to them.
![image](https://github.com/MonoGame/MonoGame/assets/154005560/0ed36772-23c3-4dfe-be52-ffecf24f2108)

Furthermore, in the C# scripts, the 'Microsoft.Xna' and 'Android' namespaces could not be found.
![image](https://github.com/MonoGame/MonoGame/assets/154005560/a4e6452b-7a0d-4469-83ed-ea881f21c5d1)

I have researched this constantly, but I have never found an answer that actually works.
Does anyone have any idea?

Thank you in advance!

**Comment by dellis1972** [comment]
Did you update your project to target net8.0-android ? 
Do you have the android workload installed? (`dotnet workload install android`).
